crates/subspace-farmer/src/ws_rpc_server.rs

```diff
@@ -152,6 +152,8 @@ pub struct RpcServerImpl {
     object_mappings: Arc<Vec<ObjectMappings>>,
 }
 
+// TODO: Reconstruction here is a bit incorrect: it doesn't account for source/parity interleaving
+//  and raw records
```

Comment:
It is not used already. This server was previously exposed by farmer as RPC, but not used for a long time since farmer is no longer storing all the data locally. This is for the most part sitting here before being moved under `subspace-archiving` and rewritten as a reconstructor, similar to those we already have for blocks and pieces, but for objects. Given the fact that objects are in unknown state, I didn't bother doing anything about it yet.